client/web/src/enterprise/codeintel/configuration/pages/CodeIntelConfigurationPolicyPage.tsx

```diff
@@ -189,25 +193,31 @@ export const CodeIntelConfigurationPolicyPage: FunctionComponent<CodeIntelConfig
                     {
                         text: repo ? (
                             <>
-                                {policy?.id === '' ? 'Create a new' : 'Update a'} code graph configuration policy for{' '}
+                                {policy?.id === '' ? 'Create a new' : 'Update a'}{' '}
+                                {domain === 'scip' ? 'code graph' : 'embeddings'} configuration policy for{' '}
                                 <RepoLink repoName={repo.name} to={null} />
                             </>
                         ) : (
                             <>
-                                {policy?.id === '' ? 'Create a new' : 'Update a'} global code graph configuration policy
+                                {policy?.id === '' ? 'Create a new' : 'Update a'} global{' '}
+                                {domain === 'scip' ? 'code graph' : 'embeddings'} configuration policy
                             </>
                         ),
                     },
                 ]}
                 description={
-                    <>
-                        Rules that control{indexingEnabled && <> auto-indexing and</>} data retention behavior of code
-                        graph data.
-                    </>
+                    domain === 'scip' ? (
+                        <>
+                            Rules that control{indexingEnabled && <> auto-indexing and</>} data retention behavior of
+                            code graph data.
+                        </>
+                    ) : (
+                        <>Rules that control embeddings re-indexing.</>
```

Comment:
This is probably the only mention of "re-indexing" when it comes to embeddings in this… all the others use the words up-to-date, maintain, etc.

It's tricky without changing all the other labels… perhaps:

```suggestion
                        <>Rules that control keeping embeddings up-to-date.</>
```